torchvision/datasets/imagenet.py

```diff
@@ -0,0 +1,197 @@
+import os
+import sys
+if sys.version_info[0] == 2:
+    # FIXME: I don't know if this is good pratice / robust
+    FileExistsError = OSError
+
+import shutil
+import scipy.io as sio
```

Comment:
Sure, we can do this. As stated above, I don't know if the meta information changes between the years. If we throw out the years completely, this shouldd be easy to handle.

Reply:
moving it to inside the function is ok for now

Reply:
I've put some further thoughts into the parsing of the development kit. It mainly contains two things of importance to us:
1. The meta information, which allows us to convert between the WordNet IDs and a human readable description of each class, and
2. the ground truth for the classes of the validation images.

While the meta information is always needed, the ground truth is only needed once during the preparation of the `val` folder. I've created a [this gist](https://gist.github.com/pmeier/fd31c0bcb1769fb260ec34e84e00c1ed) as an example on how this could be stored in text files. If we do not include these files (or some that contain the same information) into the package, I don't see a way to _not_ parse the development kit.